contrib/lang/asciidoc/keybindings.el

```diff
@@ -0,0 +1,2 @@
+(global-set-key (kbd "M-h") 'adoc-promote)
```

Comment:
These should not be set global

you should use something like

`(define-key adoc-mode-map (kbd "M-h") 'adoc-promote)

although the adoc mode map might be called differently